pgjdbc/src/main/java11/module-info.java

```diff
@@ -0,0 +1,18 @@
+module org.postgresql.jdbc {
+  requires static java.desktop;
+  requires java.logging;
+  requires java.management;
+  requires java.naming;
+  requires java.security.jgss;
+  requires java.sql;
+  requires java.transaction.xa;
+  requires static org.checkerframework.checker.qual;
+  requires static com.sun.jna;
+  requires static com.sun.jna.platform;
+  requires static waffle.jna;
+  requires static org.osgi.service.jdbc;
+  requires static osgi.core;
+
+  exports org.postgresql;
```

Comment:
I think we might need to declare more packages for the exports

Reply:
We could mark the whole module as open, or try to be more selective. Is there a list of packages that should be exported?